packages/schema/src/Schema.ts

```diff
@@ -5007,6 +5007,24 @@ export const DateFromString: DateFromString = transform(
   { decode: (s) => new Date(s), encode: (n) => n.toISOString() }
 ).annotations({ identifier: "DateFromString" })
 
+/**
+ * @category api interface
+ * @since 1.0.0
+ */
+export interface DateFromNumber extends Annotable<DateFromNumber, Date, number> {}
+
+/**
+ * Represents a schema that converts a `number` into a (potentially invalid) `Date` (e.g., `new Date(Infinity)` is not rejected).
+ *
+ * @category Date transformations
+ * @since 1.0.0
+ */
+export const DateFromNumber: DateFromNumber = transform(
```

Comment:
Please document whether the number represents epoch seconds, epoch milliseconds, or something else. (I would also make the same request about the documentation above for DateFromString)